x-pack/platform/plugins/shared/alerting/server/application/rule/methods/create/schemas/create_rule_data_schema.ts

```diff
@@ -39,6 +39,7 @@ export const createRuleDataSchema = schema.object(
     notifyWhen: schema.maybe(schema.nullable(notifyWhenSchema)),
     alertDelay: schema.maybe(alertDelaySchema),
     flapping: schema.maybe(schema.nullable(flappingSchema)),
+    dashboards: schema.maybe(schema.arrayOf(schema.object({ id: schema.string() }))),
```

Comment:
Something similar is defined [here](https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/alerting/common/routes/rule/apis/create/schemas/v1.ts) as well. Could these two be unified somehow?